build.gradle

```diff
@@ -0,0 +1,30 @@
+plugins {
+    id 'org.springframework.boot' version '2.6.3'
+    id 'io.spring.dependency-management' version '1.0.11.RELEASE'
+    id 'java'
+}
+
+group = 'com.elixr'
+version = '0.0.1-SNAPSHOT'
+sourceCompatibility = '17'
+
+repositories {
+    mavenCentral()
+}
+
+dependencies {
+    implementation 'org.springframework.boot:spring-boot-starter-data-mongodb'
+    implementation 'org.springframework.boot:spring-boot-starter-web'
+    implementation 'com.jamierf.db-table:db-table-core:0.1'
```

Comment:
What is this used for?

Reply:
previously I tried to connect docker with mongo DB (just to try), forgot to remove dependency.